.github/workflows/release.yml

```
@@ -13,15 +13,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Install Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-          registry-url: "https://registry.npmjs.org"
       - name: Install pnpm
         uses: NullVoxPopuli/action-setup-pnpm@v2
         with:
           pnpm-version: 8.5.1
+          node-version: 16
+          registry-url: "https://registry.npmjs.org"
```

The argument should be `node-registry-url` [here](https://github.com/NullVoxPopuli/action-setup-pnpm#node-registry-url)
```suggestion
          node-registry-url: "https://registry.npmjs.org"
```